tests/bsim/bluetooth/host/adv/encrypted/css_sample_data/testcase.yaml

```
@@ -0,0 +1,10 @@
+tests:
+  bluetooth.host.adv.encrypted.css_sample_data:
+    build_only: true
+    tags:
+      - bluetooth
+    platform_allow:
+      - nrf52_bsim
```

@aescolar should this also be?

```suggestion
      - nrf52_bsim/native
```

---

It does not matter as it is the only option.